README.md

```diff
@@ -1,5 +1,4 @@
 # Web Fundamentals on DevSite
-
```

Comment:
Please leave the blank lines in, our MD processor can be picky at times are requires them.